src/sqs/discovery.ts

```diff
@@ -1,12 +1,18 @@
-import {MessageHandler} from "./index"
+import { MessageHandler } from "./index";
 import app from "../worker/app";
-import {discovery} from "../sync/discovery";
+import { discovery, discoveryOctoKit } from "../sync/discovery";
+import { booleanFlag, BooleanFlags } from "../config/feature-flags";
 
 export type DiscoveryMessagePayload = {
 	installationId: number,
 	jiraHost: string
 }
 
 export const discoveryQueueMessageHandler : MessageHandler<DiscoveryMessagePayload> = async (context) => {
-	await discovery(app)({data: context.payload}, context.log);
-}
+	const useNewGHClient = await booleanFlag(BooleanFlags.USE_NEW_GITHUB_CLIENT_FOR_DISCOVERY, false, context?.payload?.jiraHost);
```

Comment:
/nit 

Would be great to add jiraHost / installation ID to the context.log, so when logs are logged we can see for which JiraHost and installationId.

	context.log = context.log.child({
		jiraHost: context?.payload?.jiraHost
		installationId: context?.payload?.installationId
	})

Reply:
Added, was able to remove passing jirahost/installationId around the place too..... win!!